### PR TITLE
re:casings

### PR DESCRIPTION
**Pull request by tmtmtl30**
## About The Pull Request

Changes the sprites of ammo casings (/obj/item/ammo_casing) and shotgun shells. In addition to a number of specialized sprites, standard bullets now have a variety of skins based on their size (pistol, magnum, rifle, or Big), skin (brass casing or steel casing), and bullet variant (lead surplus, blue rubber, white AP, red hollow-point, and orange incendiary). To make adding the new sprites less excruciating, the dirs on casing sprites have been removed in favor of rotating the sprites through transforms.

Also, I changed the typepath of scatter laser shells from laserslug to laserscatter, because they aren't slugs.

- [X] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game

shit looks cash
image
(note: the pellets don't rotate anymore, because i realized they looked dumb that way. also ignore the quiver)
image

## Changelog

:cl: TetraZeta, tmtmtl30
imageadd: new ammo casing sprites, courtesy of TetraZeta
tweak: ammo casing sprites moved to their own file
tweak: shotgun shell sprites moved to their OWN own file
tweak: ammo casings now automatically rotate using transforms instead of dirs
/:cl:

### CODE REVIEW

File: code/datums/components/crafting/recipes.dm [hunk 1]
```diff
@@ -318,9 +318,9 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_AMMO
 
-/datum/crafting_recipe/laserslug
+/datum/crafting_recipe/laserscatter
 	name = "Scatter Laser Shell"
-	result = /obj/item/ammo_casing/shotgun/laserslug
+	result = /obj/item/ammo_casing/shotgun/laserscatter
```
Comment (triplezeta) [inline code/datums/components/crafting/recipes.dm:323]: make sure to repath this in ruins

Comment (tmtmtl30) [reply code/datums/components/crafting/recipes.dm:323]: oh yeah i forgot i had .dmms blocked from search in VSC

**Review by triplezeta — APPROVED** [review]
wow these sprites are so good!! whoever made them must be so talented

**Comment by TallDeekus** [comment]
> wow these sprites are so good!! whoever made them must be so talented

Get over here and accept my compliments you m.

**Comment by TallDeekus** [comment]
Merged by the person who made the sprites, kinda cringe >:)

**Pull request merged**